### PR TITLE
add issue template

### LINKED ISSUE #156: Adding issue template
A issue template can be added so that creating new issues for the beginner would be easy

Please assign this issue to me so that I can start working on it

### PR DESCRIPTION
**Pull request by shreyaansjain06**
Now creating issue will be easier for contributors
fixes #156

Please consider it under hactoberfest

### CODE REVIEW

**Comment by GuillemPM** [comment]
I think it would be nice to create the template using YAML instead of Markdown, as it is more intuitive to use.
See an example using YAML:



Code example:
```yaml
name: 🐞 Bug Report
description: Something does not work or is flaky! let us know!
labels: [bug, triage]
body:
  - type: input
    id: version
    attributes:
      label: Project Version
      placeholder: e.g. vX.Y.Z
    validations:
      required: true

  - type: input
    id: os_version
    attributes:
      label: Operating System (or Browser)
    validations:
      required: true

  - type: input
    id: node_version
    attributes:
      label: Node Version (if applicable)
    validations:
      required: false
...
```

if @origranot agrees, could you update your PR to follow this format?

**Comment by GuillemPM** [comment]
@shreyaansjain06, you can begin doing the changes, since we all agree 👍

**Comment by shreyaansjain06** [comment]
I have made the changes please review it

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: 🐛 Bug
+description: Report an issue to help improve the project.
+labels: ['🛠 goal: fix']
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: Let's use basic tags

```suggestion
labels: [bug, triage]
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: '[DOCS] <description>'
+labels: ['📄 aspect: text']
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/docs.yml:4]:
```suggestion
labels: [docs, triage]
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for URL Shortner? Please suggest!
+title: '[FEATURE] <description>'
+labels: ['⭐ goal: addition']
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/feature_request.yml:4]:
```suggestion
labels: [enhancement, triage]
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER]'
+labels: ['🚦 status: awaiting triage']
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/other.yml:4]:
```suggestion
labels: [other, triage]
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: '[DOCS] <description>'
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/docs.yml:3]:
```suggestion
title: '[DOCS]'
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,26 @@
+name: 💡 General Feature Request
+description: Have a new idea/feature for URL Shortner? Please suggest!
+title: '[FEATURE] <description>'
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/feature_request.yml:3]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 7]
```diff
@@ -0,0 +1,22 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER]'
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/other.yml:3]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 8]
```diff
@@ -0,0 +1,27 @@
+name: 📄 Documentation issue
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/docs.yml:1]:
```suggestion
name: 📄 Documentation suggestion
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+description: Found an issue in the documentation? You can use this one!
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/docs.yml:2]:
```suggestion
description: Suggestions on how we can improve the documentation.
```

File: .github/ISSUE_TEMPLATE/docs.yml [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+    attributes:
+      label: Screenshots
+      description: Please add screenshots if applicable
+    validations:
+      required: false
+  - type: textarea
+    id: extrainfo
+    attributes:
+      label: Additional information
+      description: Is there anything else we should know about this issue?
```
Comment (GuillemPM) [inline .github/ISSUE_TEMPLATE/docs.yml:24]:
```suggestion
      description: Is there anything else we should know about this suggestion?
```

**Review by GuillemPM — CHANGES_REQUESTED** [review]
Thank you so much! They look pretty good ♥

I have made some changes that fit our project better, please accept them and I will merge this.

**Pull request merged**